apps/designer/package.json

```diff
@@ -37,6 +37,7 @@
     "@trpc/server": "^10.9.0",
     "@vercel/node": "^2.4.1",
     "@webstudio-is/asset-uploader": "workspace:^",
+    "@webstudio-is/authorization-token": "workspace:^",
```

Comment:
why not call it authorization? are you afraid unrelated stuff would go in?

Reply:
Yep, as of now oss authorization logic is inside trpc-interface